CHANGELOG.md

```diff
@@ -8,9 +8,11 @@ and this project (partially) adheres to [Semantic Versioning](https://semver.org
 ## [Unreleased]
 ### Added
 - Official EndeavourOS distribution support
+- Display server protocol to `WindowManager`
 
 ### Changed
 - Allow `df` output to contain Unicode characters
+- `WindowManager` API value format (now an object)
```

Comment:
```suggestion
- `WindowManager` API value format: now an object with `name` and `display_server_protocol` attributes
```

Worth being more specific?

Reply:
Yes, it would be better. I wonder whether DSP should rather be added to `DestktopEnvironment` entry though 🤔